modules/consensus.go

```diff
@@ -30,9 +30,22 @@ type (
 	ConsensusSetSubscriber interface {
 		// ProcessConsensusChange sends a consensus update to a module through
 		// a function call. Updates will always be sent in the correct order.
+		// There may not be any reverted blocks, but there will always be
+		// applied blocks.
 		ProcessConsensusChange(ConsensusChange)
 	}
 
+	// ConsensusSetDigestSubscriber receives digests about changes to the
+	// consensus set in the form of a list of ids of blocks that got reverted
+	// and a list of ids of blocks that got applied. The ordering is always all
+	// of the reverted blocks followed by all of the applied blocks.
```

Comment:
what is meant by "ordering" here? Aren't the reverted/applied blocks two separate arguments?